src/Library/demos/Row/main.blp

```diff
@@ -0,0 +1,104 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Rows";
+  description: "Create user interface lists with ease";
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+      Adw.PreferencesGroup  {
+
+      title: "Rows can be groupped";
+
+      Adw.ActionRow {
+        title: "Action row can have an icon";
+        icon-name: "info";
+      }
```

Comment:
row icon is deprecated

Alex explained here: https://matrix.to/#/!kDBZrVKCdhrVuWxbGe:matrix.org/$nWquPXRbvc9ohJRBvFARZZpSnC9JXIhLtRIJkRSWHl8?via=gnome.org&via=matrix.org&via=catgirl.cloud